Apps/WebClient/src/ClientApp/src/components/modal/noteEdit.vue

```diff
@@ -247,10 +238,7 @@ export default class NoteEditComponent extends Vue {
             <b-row class="w-100 h-100">
                 <b-col cols="auto">
                     <div class="icon">
```

Comment:
nit: these should be renamed to icon-wrapper or something along those lines since they are not actually classes for icons, but for the wrappers for them

Reply:
Fixed in latest commit.